documentation/Readme.md

```diff
@@ -0,0 +1,47 @@
+# Project Title
+- Cool logo
+
+---
+
+### Introduction
+
+- Summary of the purpose and function of this project
+- Provide links to Issue queue and project page (if any)
+
+### Requirements
+
+- Hardware requirements (Processor/Storage/RAM/GPU)
+- OS requirements (List of compatible Operating Systems)
+- List of Dependencies
+
+### Installation
+
+- Guide/code to install/build dependencies for respective OS
+- If a dependency is to be built, provide a link to its *Installation* page. Do not reinvent the wheel here.
+- Instructions to build the project after the dependencies have been met
+
+### Configuration
+
+- List any configuration is required; for example, the various config options in cmake.
+- Any special notes about the configuration of this module – including but not limited to permissions.
+
+### Testing
+
+- Instructions to run unit tests.
+- Each unit test should have internal documentation on what the test is checking for, and the logic that is being used.
+
+- Instructions for Integration Testing
+
+- Instructions to follow when tests fail. (This may also be included int eh Troubleshooting section)
+
+### Troubleshooting and FAQ
+This sections builds up over time.
+
+- List out common problems that users face, along with solutions.
+- What to do when something breaks.
+
+### Maintainers
+
+- Names of people associated with the project split into two lists :
```

Comment:
Along with names, we should also add an email address of the member.

Reply:
Yes, I agree.